Examples/Goals.md

```diff
@@ -0,0 +1,3 @@
+* Get rid of both of your instruments.
+* Play a consonance for at least 4 seconds.
+* Eat a sandwich.
```

Comment:
I am not sure if this goal is achievable

Reply:
Oh yes, maybe I change this goal.